test/src/main/configurations/MainConfig/ConfigurationUnzip.xml

```diff
@@ -6,9 +6,16 @@
 			serviceName="ibis4test-ZipFilePathListener" />
 	</Receiver>
 
-	<Pipeline firstPipe="unzipFile">
+	<Pipeline firstPipe="createOutputFolder">
```

Comment:
if the firstPipe is the first pipe, it is not necessary to configure it, so then you can remove the attribute